ci-operator/config/openshift/image-customization-controller/openshift-image-customization-controller-main.yaml

```diff
@@ -1,20 +1,37 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_builder_rhel-8-golang-1.16-openshift-4.10:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.10
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.16
+  from_repository: true
+images:
+- from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.10:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10
+  to: image-customization-controller
 promotion:
-  name: "4.9"
+  name: "4.10"
   namespace: ocp
```

Comment:
If we remove this "promotion" section it will eliminate the part of the patch that modifies the mirror list, which needs approval from Clayton. That will get tests of the image build working, and we can follow up with a patch that does just the promotion/mirroring later.

Reply:
will do

Reply:
I had to change everything to 4.9

Reply:
promotion section is still there. You have to remove it and then run `make openshift-image-mirror-mappings jobs` to fix the generated files. See https://github.com/openshift/release/pull/23177/files for an example.